### ISSUE TITLE
Replace all audacity logos with the new one

### ISSUE THREAD

**Issue by IshoAntar** [AudMonkey/AudMonkey#5]
All audacity logos need to be replaced with the new (temporary)
logo found under `images/`.

**Comment by PracticalMetal** [comment]
Hi, I would like to work on this, can you suggest me ways to get started?

**Comment by IshoAntar** [comment]
Hi @PracticalMetal,
Thanks for your interest in contributing to AudMonkey,

There are multiple ways that could get you started, some examples:

- There are still multiple places in the codebase that link
to audacityteam.org. In the rebranding efforts, these should be modified
to point to AudMonkey's website at audmonkey.github.io (for now!).

  > Note that the above applies when the link is to the official website, audacityteam.org,
  Not the documentation/wiki, or the forums. Those should be pointed somewhere else (see below)

- Links to the support forum (forum.audacityteam.org) should be pointed
  to this project's [Discussions](https://github.com/AudMonkey/AudMonkey/discussions) (?)

- Links to the wiki (wiki.audacityteam.org) should point to audmonkey.github.io.

- `audacity` -> `audmonkey`
- `Audacity` -> `AudMonkey`
- `AUDACITY` -> `AUDMONKEY`

TL;DR swap each link with it's equivalent AudMonkey's link,

- Bug reporting -> GitHub issues
- Support/Forums -> Discussions
- Documentation -> audmonkey.github.io (although it's still empty!)
- ...

However, I'm still not sure what to do with links that point to specific
pages in Audacity's documentation, e.g. the list of [Known Issues](https://github.com/AudMonkey/AudMonkey/blob/7a696e3cfd2d7bc45dfa7db0327e6c3da5ac6655/help/audmonkey.1#L163)
and similar links that we don't have available now?
Maybe we can provide similar pages in AudMonkey's website?

Please feel free to ask if you have any questions.

**Comment by PracticalMetal** [comment]
Ok, I will be working on this
